### PR TITLE
Added UNIs to attributes_requiring_redeploy

### LINKED ISSUE #249: Changing UNIs should require redeploy
Hi,

Currently, the UNIs are not part of the `EVCBase.attributes_requiring_redeploy`. Thus, the circuit will be useless if someone sends a patch to change the UNIs by moving the circuit to another interface or even changing the UNI switch.

A workaround exists: force a redeploy right after the PATCH request (`curl -H 'Content-type: application/json' -X PATCH http://127.0.0.1:8181/api/kytos/mef_eline/v2/evc/d5aa751430264a/redeploy`)

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #249 

### Summary
Added `uni_a` and `uni_z` to `attributes_requiring_redeploy`

### Local Tests
Updated local test `test_expected_requiring_redeploy_attributes()` adding `uni_a` and `uni_z` to expected values.

### End-to-End Tests
e2e tests on PR [#201](https://github.com/kytos-ng/kytos-end-to-end-tests/pull/201)

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
@Alopalao, nicely done. Looks good to me. Could you also update the changelog?

FYI, for upcoming PRs [checkout this PR template content](https://kytos-ng.github.io/procedures/prs.html#submitting-a-pr) that our team is trying to standardize to facilitate for reviewers. In the future, we might also provide an autofill on GH.

**Review by viniarck — COMMENTED** [review]
@Alopalao just to confirm that was intentional in the changelog, is this fix expected to land as a patch for `2022.3.1` or unreleased? If it needs to land as `2022.3.1`, we'll also need to bump the version in the `kytos.json` file.

**Comment by Alopalao** [comment]
> @Alopalao just to confirm that was intentional in the changelog, is this fix expected to land as a patch for `2022.3.1` or unreleased? If it needs to land as `2022.3.1`, we'll also need to bump the version in the `kytos.json` file.

Yep, I was just following the naming criteria from previous versions. Changing it back.

File: models/evc.py [hunk 1]
```diff
@@ -41,7 +41,9 @@ class EVCBase(GenericEntity):
         "queue_id",
         "sb_priority",
         "primary_constraints",
-        "secondary_constraints"
+        "secondary_constraints",
+        "uni_a",
+        "uni_z"
```
Comment (italovalcy) [inline models/evc.py:46]:
```suggestion
        "uni_z",
```

Minor suggestion to make it easy in an eventual next modification.

File: tests/unit/models/test_evc_base.py [hunk 2]
```diff
@@ -37,7 +37,9 @@ def test_expected_requiring_redeploy_attributes(self) -> None:
             "queue_id",
             "sb_priority",
             "primary_constraints",
-            "secondary_constraints"
+            "secondary_constraints",
+            "uni_a",
+            "uni_z"
```
Comment (italovalcy) [inline tests/unit/models/test_evc_base.py:42]:
```suggestion
            "uni_z",
```

Minor suggestion to make it easy in an eventual next modification.

**Review by italovalcy — APPROVED** [review]
Approved with two minor suggestions! Very well done, Aldo!

**Review by viniarck — APPROVED** [review]

**Pull request merged**